src/Ledger/Ledger/Properties.agda

```diff
@@ -128,145 +133,136 @@ isGADeposit dp = isGADepositᵇ dp ≡ true
 govDepsMatch : LState → Set
 govDepsMatch ⟦ utxoSt , govSt , _ ⟧ˡ =
   filterˢ isGADeposit (dom (utxoSt .deposits ))
-  ≡ᵉ fromList (map (λ (id , _) → GovActionDeposit id) govSt)
+  ≡ᵉ fromList (map f govSt)
   where open UTxOState using (deposits)
 
-V⊎P∖V→DepPurposes : {txid : TxId} → List (GovVote ⊎ GovProposal) → List DepositPurpose
-V⊎P∖V→DepPurposes [] = []
-V⊎P∖V→DepPurposes {txid} (inj₁ v ∷ vps) = V⊎P∖V→DepPurposes{txid} vps -- omit votes
-V⊎P∖V→DepPurposes {txid} (inj₂ p ∷ vps) = GovActionDeposit (txid , length vps) ∷ (V⊎P∖V→DepPurposes{txid} vps)
-
 instance _ = +-0-monoid
 
+
 module _
+
+  {gΓ : GovEnv}
+
   -- ASSUMPTIONS (TODO: eliminate these) --
-  {filterGA : ∀ {txid} {n} → filterˢ isGADeposit ❴ GovActionDeposit (txid , n) ❵
-                              ≡ᵉ ❴ GovActionDeposit (txid , n) ❵ }
-  {filterCD : ∀ {c} {pp} → filterˢ isGADeposit (dom ((certDeposit c {pp})ˢ)) ≡ᵉ ∅}
-  {filterCR : (c : DCert) {deps : DepositPurpose ⇀ Coin}
-              → filterˢ isGADeposit (dom ( deps ∣ certRefund c ᶜ ˢ ))
-                ≡ᵉ filterˢ isGADeposit (dom (deps ˢ))}
-  {dom-single : {d : DepositPurpose} {c : Coin} → ❴ d ❵ˢ ≡ dom ❴ d , c ❵ˢ }
-  {ΔgovSt≡ : {txid : TxId} {govSt govSt' : GovState} {txb : TxBody}
-             → fromList (map (λ (id , _) → GovActionDeposit id) govSt')
-               ≡ᵉ fromList (map (λ (id , _) → GovActionDeposit id) govSt)
-                  ∪ fromList (V⊎P∖V→DepPurposes{txid} (txgov txb))}
+  {- 1 -} {dom-single : {d : DepositPurpose} {c : Coin} → ❴ d ❵ˢ ≡ dom ❴ d , c ❵ˢ }
```

Comment:
is the intention to leave these ones for this PR?

Reply:
Now that I'm more familiar with the ingredients involved, and have proved a few supporting lemmas about singletons and their domains, I am able to eliminate that assumption.  I will look again at the remaining assumptions and see if I can dispense with those as well; if not, then we'll leave them for a future PR.